### ISSUE TITLE
After upgrade dependencies routes stop work

### ISSUE THREAD

**Issue by marcinstasiak** [AngelMunoz/nesting-nuxt#3]
After upgrade dependencies routes stop work, like in your [Nestjs issue](https://github.com/nestjs/nest/issues/119).

```  "dependencies": {
    "@nestjs/common": "^4.5.9",
    "@nestjs/core": "^4.5.10",
    "@nestjs/typeorm": "^2.0.0",
    "@nuxtjs/axios": "^5.0.0",
    "bulma": "^0.6.2",
    "nuxt": "^1.3.0",
    "rxjs": "^5.5.6",
    "typeorm": "^0.1.12"
  },
  "devDependencies": {
    "@types/node": "^9.4.1",
    "node-sass": "^4.7.2",
    "sass-loader": "^6.0.6",
    "ts-node": "^4.1.0",
    "tslint": "^5.9.1",
    "typescript": "^2.7.1"
  }```

**Comment by AngelMunoz** [comment]
I’ll try to take a look later, I had an issue recently in my sails-nuxt repo as well, but that was fixed on subsecuent releases of nuxt, perhaps something similar will happen here,I’ll take a look and ping you back If I find something 

Saludos! -Angel D. Munoz

De: marcinstasiak
Enviado: Thursday, February 8, 2018 2:20 AM
Para: AngelMunoz/nesting-nuxt
CC: Subscribed
Asunto: [AngelMunoz/nesting-nuxt] After upgrade dependencies routes stop work(#3)

After upgrade dependencies routes stop work, like in your Nestjs issue.
    "@nestjs/common": "^4.5.9",
    "@nestjs/core": "^4.5.10",
    "@nestjs/typeorm": "^2.0.0",
    "@nuxtjs/axios": "^5.0.0",
    "bulma": "^0.6.2",
    "nuxt": "^1.3.0",
    "rxjs": "^5.5.6",
    "typeorm": "^0.1.12"
  },
  "devDependencies": {
    "@types/node": "^9.4.1",
    "node-sass": "^4.7.2",
    "sass-loader": "^6.0.6",
    "ts-node": "^4.1.0",
    "tslint": "^5.9.1",
    "typescript": "^2.7.1"
  }```

**Comment by twhitson** [comment]
Same issue here, would love to use this template.

**Comment by AngelMunoz** [comment]
Yes I've been working on this but no luck so far, I'm sure we will have to open an issue on the Nest repo

**Comment by AngelMunoz** [comment]
@marcinstasiak did you try to use a express instance?
like this
```js
async function start() {
  const instance = new Express(); // <-- like this
  const config = require('../nuxt.config');
  config.dev = !(instance.env === 'production');
  const nuxt = await new Nuxt(config);

  const app = NestFactory.create(ApplicationModule, instance);
  app.listen(3000, () => console.log('Application is listening on port 3000.'));

  instance.use(nuxt.render); // <-- like this
}
```

**Comment by twhitson** [comment]
@AngelMunoz I tried that with no success.

I forked this project, upgraded all dependencies, and got Nuxt and Nest working together with help from nestjs/nest#298. However the Nuxt `/users` route broke with the changes. I think this might be because there's a `/users` route in both Nuxt and Nest. Any further ideas?

**Comment by AngelMunoz** [comment]
Does your nestjs controller points to /api/users


? 


Because that's where axios looks for




Get Outlook for Android







On Mon, Feb 12, 2018 at 11:49 AM -0600, "Trey" <notifications@github.com> wrote:












@AngelMunoz I tried that with no success.


I forked this project, upgraded all dependencies, and got Nuxt and Nest working together with help from nestjs/nest#298. However the Nuxt /users route broke with the changes. I think this might be because there's a /users route in both Nuxt and Nest. Any further ideas?

**Comment by twhitson** [comment]
Look at the recent commits on my fork. I set the globalPrefix to `/api` and changed the controller's route to `/users`. With this configuration, both `localhost:3000` and `localhost:3000/api/users` work as expected, however `localhost:3000/users` does not.

**Comment by hawkend** [comment]
I've written my methods in issues of nestjs repostory. It may be useful for you. 
[Workaround](https://github.com/nestjs/nest/issues/414)
